tests/pages/remoteInstancesPage.js

```diff
@@ -183,6 +184,15 @@ module.exports = {
         I.fillField(this.fields.portNumber, '42200');
         I.fillField(this.fields.environment, 'remote-external-service');
         I.fillField(this.fields.cluster, 'remote-external-cluster');
+        break;
+      case 'doNotTrack':
```

Comment:
I don't think we need doNotTrack, please think of better naming for helping in easy debugging during failures and reports.